src/com/android/launcher3/DeviceProfile.java

```diff
@@ -484,11 +484,13 @@ private void updateAvailableFolderCellDimensions(DisplayMetrics dm, Resources re
     }
 
     private void updateFolderCellSize(float scale, DisplayMetrics dm, Resources res) {
+        int folderLabelRowCount = prefs.getHomeLabelRows();
```

Comment:
There was an issue here that `prefs` wasn't a member in this class yet, to get rid of all our instantiations of this all over the class I fixed this and applied your commit internally.

Reply:
Oh right. I just followed the format in the workspace and all apps sections, I should've known I missed it out.
The fix was to add `LawnchairPreferences prefs = Utilities.getLawnchairPrefs(mContext);` correct?

Reply:
Yup that would have fixed it!